src/templates/userRetention.mjml

```diff
@@ -41,65 +38,90 @@
           <mj-text>
             {{/equals}}
           </mj-text>
-          <mj-spacer height="24px" />
-          <mj-text mj-class="color-grey-darker">
+          <mj-text>
             {{#greaterThan (length @root.articlesRecommended) 0}}
-              <mj-column>
-                {{#each @root.articlesRecommended}}
-                  <a class="article-digest link-black weight-medium article-recommended" href="{{@root.siteDomain}}/@-/-{{this.mediaHash}}">
-                    <span class="article-digest-title">{{this.title}}</span>
-                    <span class="article-digest-subtitle">{{this.displayName}}</span>
-                  </a>
-                {{/each}}
-              </mj-column>
+          </mj-text>
+          <mj-text mj-class="color-grey-darker">
+            ${{userRetention.newcomer.p3}}$
+          </mj-text>
+          <mj-spacer height="24px" />
+          <mj-text>
+            <mj-column>
+              {{#each @root.articlesRecommended}}
+                <a class="article-digest link-black weight-medium article-recommended" href="{{@root.siteDomain}}/@-/-{{this.mediaHash}}">
+                  <span class="article-digest-title">{{this.title}}</span>
+                  <span class="article-digest-subtitle">{{this.displayName}}</span>
+                </a>
+              {{/each}}
+            </mj-column>
+          </mj-text>
+          <mj-text>
             {{/greaterThan}}
+          </mj-text>
+          <mj-text>
             {{#greaterThan (length @root.articlesHottest) 0}}
-              <mj-column>
-                {{#each @root.articlesHottest}}
-                  <a class="article-digest link-black weight-medium article-recommended" href="{{@root.siteDomain}}/@-/-{{this.mediaHash}}">
-                    <span class="article-digest-title">{{this.title}}</span>
-                    <span class="article-digest-subtitle">{{this.displayName}}</span>
-                  </a>
-                {{/each}}
-              </mj-column>
-            {{/greaterThan}}
           </mj-text>
-          <mj-spacer height="32px" />
+          <mj-text mj-class="color-grey-darker">
+            ${{userRetention.newcomer.p4}}$
+          </mj-text>
+          <mj-spacer height="24px" />
+          <mj-text>
+            <mj-column>
+              {{#each @root.articlesHottest}}
+                <a class="article-digest link-black weight-medium article-recommended" href="{{@root.siteDomain}}/@-/-{{this.mediaHash}}">
+                  <span class="article-digest-title">{{this.title}}</span>
+                  <span class="article-digest-subtitle">{{this.displayName}}</span>
+                </a>
+              {{/each}}
+            </mj-column>
+          </mj-text>
+          <mj-spacer height="40px" />
           <mj-divider border-color="#F7F7F7"></mj-divider>
```

Comment:
same as above, dividers should be placed at the top of the conditional block instead of bottom?

Reply:
otherwise, it's not self-contained